src/veritysetup/veritysetup.c

```diff
@@ -104,8 +106,19 @@ static int parse_options(const char *options) {
                 else if (streq(word, "panic-on-corruption"))
                         arg_activate_flags |= CRYPT_ACTIVATE_PANIC_ON_CORRUPTION;
 #endif
-                else if ((val = startswith(word, "root-hash-signature="))) {
+                else if ((val = startswith(word, "hash-offset="))) {
+                        uint64_t off;
+
+                        r = parse_size(val, 1024, &off);
+                        if (r < 0)
+                                return log_error_errno(r, "Failed to parse offset '%s': %m", word);
+                        if (off % 512)
```

Comment:
doesn't matter much, but in systemd we have this rule of only relying on C's downgrade-to-bool feature for stuff that is either boolean or boolean-like, or a pointer. But for arithmetic values we'd rather have people do comparisons explicitly, i.e. here `if (off % 512 != 0)`.

See https://systemd.io/CODING_STYLE/#using-c-constructs

Reply:
done